### PR TITLE
cmd-ref: make changes related to update --rev

### LINKED ISSUE #1062: cmd-ref: address reviews from #1007
- [x] https://github.com/iterative/dvc.org/pull/1007#pullrequestreview-360472892
- [ ] https://github.com/iterative/dvc.org/pull/1007#pullrequestreview-367607133

### PR DESCRIPTION
**Pull request by skshetry**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please chose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

Fix #1062. @jorgeorpinel, not all suggestions have been addressed though. :slightly_smiling_face:

### CODE REVIEW

File: content/docs/command-reference/update.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ dvc update --rev master
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-## Example: Updating imported artifacts
+## Example:
```
Comment (shcheklein) [inline content/docs/command-reference/update.md]: better keep name (or remove `:`)

Comment (skshetry) [inline content/docs/command-reference/update.md]:
```suggestion
## Example
```

**Comment by shcheklein** [comment]
@jorgeorpinel please review

**Comment by shcheklein** [comment]
@jorgeorpinel ping!

**Comment by skshetry** [comment]
@jorgeorpinel, I have left two suggestions: one related to changing examples (to show about moving between tag to a latest rev in master), and other related to renaming *revision* to *commit*.

For former one, I don't want to modify current example, but feel free to add one if you think it's important. For later one, *commit* is not really an accurate word to use here for `rev`, because it really is a revision/reference, so, it can be anything like branch name, short-SHA, tag, etc.

**Comment by jorgeorpinel** [comment]
@skshetry I think we're talking about different reviews. There were 4 pending comments in https://github.com/iterative/dvc.org/pull/1007#pullrequestreview-367607133 which don't seem to match with your comments. Anyway, I'm applying them in f8789da6 for #1110

**Pull request merged**